<launch>

	<node name="theatre_uav_trajectory_planner" pkg="theatre_uav_trajectory_planner" type="trajectory_planner" output="screen">
		<param name="rate" value="10.0"/>
		<param name="duration" value="20.0"/>
		<param name="x_square" value="0.75"/>
		<param name="y_square" value="0.4"/>
		<param name="altittude" value="1.3"/>
	</node>


</launch>
